test/deploy.sh

```diff
@@ -24,6 +24,7 @@ function cleanup() {
    done
    # Remove bc_config.json before starting experiment.
    rm -f bc_config.json
+   rm -rf /tmp/harmony*
```

Comment:
is the default path of the db in /tmp? 

We should make it in a dedicated directory like $HOME  or just current directory because /tmp is usually mounted as tmpfs, which is still memory based file system.

Reply:
done.